connectors/es/sink.py

```diff
@@ -854,6 +883,7 @@ async def cancel(self):
             self._sink_task.cancel()
         if self._extractor_task_running():
             self._extractor_task.cancel()
+        self.canceled = True
```

Comment:
Is it used for anything? I only see assignments in this PR, but not usage

Reply:
it's used here: https://github.com/elastic/connectors/pull/2364/files#diff-f4def1564167603a89d66578a9ac61c61f5d0c63298e1ceab10b22f3d949ff8eR347